ScreenRuler/RulerPainter.cs

```diff
@@ -116,34 +124,53 @@ public void PaintMarkers(Graphics g, float mouseLine, LinkedList<int> customLine
             if (settings.ShowCenterLine)
             {
                 float pos = (float)rulerLength / 2;
-                drawLine(g, size, settings, pos,
-                    settings.Theme.CenterLineColor, converter);
+                Color col = settings.Theme.CenterLineColor;
+                drawLine(g, size, settings, MarkerSymbolCenterLine, pos, col, converter);
             }
             // Draw line showing the position of the cursor
             if (settings.ShowMouseLine)
             {
-                drawLine(g, size, settings, mouseLine,
-                    settings.Theme.MouseLineColor, converter);
+                Color col = settings.Theme.MouseLineColor;
+                drawLine(g, size, settings, MarkerSymbolMouseLine, mouseLine, col, converter);
             }
             // Draw the lines showing the thirds of the ruler
             if (settings.ShowThirdLines)
             {
                 float third = (float)rulerLength / 3;
                 Color col = settings.Theme.ThirdsLinesColor;
-                drawLine(g, size, settings, third, col, converter);
-                drawLine(g, size, settings, 2 * third, col, converter);
+                drawLine(g, size, settings, MarkerSymbolThirdLines, third, col, converter);
+                drawLine(g, size, settings, MarkerSymbolThirdLines, 2 * third, col, converter);
+            }
+            // Draw the line showing the Golden Ratio  
+            // Golden Ratio: A/B = (A+B)/A, where A > B > 0
+            // The marker | shows A of Golden ratio: <---A---|-B->
+            // Add a second marker showing same thing but from right/bottom instead, called B
+            if (settings.ShowGoldenLine)
+            {
+                float goldenA = phi * rulerLength;
+                float goldenB = rulerLength - goldenA;
+                Color col = settings.Theme.GoldenLineColor;
+                drawLine(g, size, settings, MarkerSymbolGoldenLine, goldenB, col, converter);
+                drawLine(g, size, settings, MarkerSymbolGoldenLine, goldenA, col, converter);
             }
             // Draw all given custom markers
+            int x = 0;
             foreach (int line in customLines)
             {
-                drawLine(g, size, settings, line, settings.Theme.CustomLinesColor, converter);
+                // Symbols for 1-20 using unicode. More than 20 - skip symbol
+                var symbol = x < 20 ? char.ConvertFromUtf32(MarkerSymbolCustomLine1 + x++) : $"{x++} ";
```

Comment:
Good find! That's a case I missed.  I figured it's not likely to go that high (it will get rather clogged with that many markers) and just added something quickly to cover that case anyway. 
One option would be to start with the unicode char preceding circled one and counter x at 1. I truly enjoy one-liners but X++ at two places was not so pretty. I'll fix a working solution to that nevertheless.